File: src/main/resources/output/IFC4X3_csharp/Entities/IfcRoot/IfcObjectDefinition/IfcObject/IfcProduct/IfcAnnotation/IfcAnnotation.cs
public class IfcAnnotation : IfcProduct
{
    public IfcAnnotationTypeEnum PredefinedType { get; set; }

    // === INVERSE CLAUSES ===
    // ContainedInStructure : SET [0:1] OF IfcRelContainedInSpatialStructure FOR RelatedElements
}

File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcRoot/IfcObjectDefinition/IfcObject/IfcProduct/IfcAnnotation/IfcAnnotation.vb
Public Class IfcAnnotation Inherits IfcProduct
    Public PredefinedType As IfcAnnotationTypeEnum

    ' === INVERSE CLAUSES ===
    ' ContainedInStructure : SET [0:1] OF IfcRelContainedInSpatialStructure FOR RelatedElements
End Class
